docker/Dockerfile.multiarch

```diff
@@ -0,0 +1,22 @@
+FROM --platform=$BUILDPLATFORM golang:1.16 AS build
+
+RUN apt-get install make
+WORKDIR /src
+COPY . .
+ARG TARGETOS TARGETARCH
+RUN --mount=type=cache,target=/root/.cache/go-build \
+    --mount=type=cache,target=/go/pkg \
+    make build
+
+FROM alpine:3.15
+RUN apk add --no-cache ca-certificates mailcap git openssh curl perl && \
+	echo "@testing http://nl.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
```

Comment:
Why do we need testing repos?

Reply:
I included them since they were in the plugin-base image.  I removed them in the latest commit.